src/client/bridge/gateway/intents.rs

```diff
@@ -5,9 +5,30 @@ use serde::{
 };
 
 /// [Gateway Intents] will limit the events your bot will receive via the gateway.
-/// By default, no intents are specified by Serenity.
+/// By default, all intents except [Privileged Intents] are selected.
 ///
-/// [Gateway Intents]: https://discordapp.com/developers/docs/topics/gateway#gateway-intents
+/// # What are Intents
```

Comment:
There should be a new line after the header.
```suggestion
/// # What are Intents
///
```